### DISCUSSION TITLE
ETA version 5 of formiojs and @formio/react

### DISCUSSION THREAD

**Discussion by e-gauthier** [formio/formio.js#5291] [General]
Is there an actual date for a ETA for both packages using version 5?

We want to upgrade our app to the latest  formiojs version (5.0.0-rc.20) but @formio/react sill use the latest version 4 and prevent us to upgrade both of them.

Thx.

**Comment by brendanbond** [comment]
Hey @e-gauthier thanks for asking. The best I can say right now is "very soon," we're in the final stages of getting features delivered, PRs merged, and testing done. Thanks for your patience.
